draft-ietf-quic-transport.md

```diff
@@ -2352,171 +2361,136 @@ shown in the following figure and described below.
                             |        |
                             +--------+
                                  |
-                        send/recv STREAM/RST
-                             recv MSD/SB
+                      send STREAM or RST_STREAM
                                  |
                                  v
-                 recv FIN/  +--------+    send FIN/
-                 recv RST   |        |    send RST
-                  ,---------|  open  |-----------.
-                 /          |        |            \
-                v           +--------+             v
-         +----------+                          +----------+
-         |   half   |                          |   half   |
-         |  closed  |                          |  closed  |
-         | (remote) |                          |  (local) |
-         +----------+                          +----------+
-             |                                        |
-             |   send FIN/  +--------+    recv FIN/   |
-              \  send RST   |        |    recv RST   /
-               `----------->| closed |<-------------'
+                            +--------+
+                            |        |
+                            |  open  |
                             |        |
                             +--------+
+                                 |
+                      send STREAM with FIN flag
+                         send RST_STREAM
+                                 |
+                                 v
+                            +--------+
+                            |        |
+                            | closed |
+                            |        |
+                            +--------+
+~~~
```

Comment:
@janaiyengar and I agreed to make the draft consistent about closing when the frame is first sent.  Based on reading #572, I think that we should revert that change.  The states are basically advisory only and the degree to which the documentation manages to capture these important notions makes it more useful to enclose rather than to leave retransmission up to the "transport".